client/js/lounge.js

```diff
@@ -111,6 +111,47 @@ $(function() {
 		index: 1
 	};
 
+	const colorCodeMap = [
+		[0, "White"],
+		[1, "Black"],
+		[2, "Blue"],
+		[3, "Green"],
+		[4, "Red"],
+		[5, "Brown"],
+		[6, "Magenta"],
+		[7, "Orange"],
+		[8, "Yellow"],
+		[9, "Light Green"],
+		[10, "Cyan"],
+		[11, "Light Cyan"],
+		[12, "Light Blue"],
+		[13, "Pink"],
+		[14, "Grey"],
+		[15, "Light Grey"],
+	];
+
+	const colorStrategy = {
+		id: "colors",
+		match: /^\x03(\d{0,2}|[A-Za-z ]{0,10})$/,
+		search(term, callback) {
+			term = term.toLowerCase();
+			const matchingColorCodes = colorCodeMap
+				.filter(i => i[0].toString().startsWith(term) || i[1].toLowerCase().startsWith(term));
+
+			callback(matchingColorCodes);
+		},
+		template(value) {
+			if (value === colorCodeMap[0]) {
```

Comment:
Agreed, but I couldn't find a good way to do that simply without having to duplicate the entire `.irc-(f|b)g*` CSS palette, which I really don't want to do.
An alternative is to... not care, and first line is going to be white text on white bg + non-white hover.
Any suggestions?